transformer/task.py

```diff
@@ -209,11 +209,13 @@ def req_to_expr(r: Request) -> py.FunctionCall:
         args["headers"] = py.Literal(headers)
 
     if r.method is HttpMethod.POST:
-        rpd = RequestsPostData.from_har_post_data(r.post_data)
-        args.update(rpd.as_kwargs())
+        if r.post_data:
```

Comment:
Why didn't any existing test fail? If it's possible, let's add one (on top of the functional one)

Reply:
There are unit tests for POST requests with urlencoded and JSON payloads, but not without any payload. I don't know the HTTP spec(s) well enough to know what is legitimate…

Reply:
The new unit tests showed other bugs! Fixed :tada: